source/user/sgx.rst

```diff
@@ -0,0 +1,52 @@
+.. _user_sgx:
+
+Seminumerical Exchange (SGX)
+****************************
+
+*Modules*: :mod:`sgx`
+
+The SGX module implements seminumerical computation of the exchange matrix.
+
+Introduction
+============
+
+Direct computation of the Hartree-Fock Exchange matrix in the atomic orbital basis scales poorly with system size. To achieve better scaling, one coordinate of the two-electron integrals is computed analytically, while the other is evaluated on a real-space grid, as proposed by Friesner :cite:`Friesner1985`. The PySCF implementation most closely resembles the chain-of-spheres (COSX) algorithm of Neese et al. :cite:`Neese2009`, but with more conservative grids and without P-junction screening. Overlap fitting is used to reduce aliasing errors :cite:`Izsak2011`. SGX scales as :math:`O(N^2)` with system size, as opposed to the :math:`O(N^4)` scaling of analytical exchange :cite:`Neese2009`.
+
+Usage and Example
+=================
+
+Any :attr:`scf.hf.SCF` object :attr:`mf` can be converted to an equivalent object that computes the Coulomb and Exchange matrices with SGX instead of analytical integration by calling :code:`sgx.sgx_fit(mf)`.
```

Comment:
```suggestion
Any :attr:`scf.hf.SCF` object :attr:`mf` can be converted to an equivalent object that computes the Coulomb and exchange matrices with SGX instead of analytical integration by calling :code:`sgx.sgx_fit(mf)`.
```